example-config.yml

```diff
@@ -68,6 +68,7 @@ persistence:
 #  enabled: true
 #  strategy: otp_middleware
 #  auth0:
+#    cacheLocation: localstorage (optional - default is "memory")
```

Comment:
Please add this new attribute to `Auth0Config` in config-types.ts.

Reply:
Make the values a type that accepts two strings, i.e. `cacheLocation: 'memory' | 'localstorage'` or the equivalent type from the Auth0 library.